src/main/java/frc/robot/commands/FeederIndexCommand.java

```diff
@@ -17,14 +17,14 @@ public void initialize() {}
     // Called every time the scheduler runs while the command is scheduled.
     @Override
     public void execute() {
-        System.out.println(Robot.CONVEYANCE_SUBSYSTEM.getConveyanceOneSubsystemHasBall());
-        System.out.println(Robot.FEEDER_SUBSYSTEM.getFeederSubsystemHasBall());
+        System.out.println(Robot.CONVEYANCE_SUBSYSTEM.getConveyanceHasBall());
```

Comment:
remove these printlns for now, we can add sensor information to a dashboard later